bazel/linux/defs.bzl

```diff
@@ -259,6 +259,16 @@ def _kernel_modules(ctx):
         modules = " ".join(modules),
     )
 
+    compilation_mode = ctx.var["COMPILATION_MODE"]
+    if compilation_mode == "fastbuild":
+        cflags = ""
+    elif compilation_mode == "opt":
+        cflags = ""
+    elif compilation_mode == "dbg":
+        cflags = "-O1 -fno-inline-functions-called-once"
```

Comment:
we hard code -g in our Makefile, should -g be provided from here instead? Maybe something like:
"fastbuild" -> -g
"opt" -> no -g
"dbg" -> -ggdb3 or whatever works best, even if more expensive.

Reply:
I tried your suggestion but it looks like that `-g` in our Makefile is already redundant. The kernel Makefile will add a `-g` depending on `CONFIG_DEBUG_INFO`: https://github.com/torvalds/linux/blob/master/scripts/Makefile.debug. I don't know if there is a way for a module to opt-out. David is not in this repository so I can't tag him for his input as well.

Reply:
In any case, I added `-g` for fastbuild and dbg.